Header header
bool target_found
geometry_msgs/Point target_position

# Number of pixels in the camera image, for scaling later
uint32 camera_width
uint32 camera_height

